core/src/validator.rs

```diff
@@ -566,7 +566,8 @@ impl Validator {
                 .map_err(|err| {
                     format!(
                         "Failed to backup and clear shreds with incorrect \
-                        shred version from blockstore: {err}"
+                        shred version from blockstore: {:?}",
+                        err
```

Comment:
clippy or fmt will probably want it like this
```suggestion
                        shred version from blockstore: {err:?}",
```

Reply:
yeah. fixed.